2020_s1/box_vagrant/README.md

```diff
@@ -6,10 +6,11 @@
 ## Building the Vagrant Enviornment
```

Comment:
Yeah you are right, the `cd` could also fails, do you mind to add the notes in the beginning:
 
```suggestion
** Windows users please use PowerShell for the below command**
```

Reply:
@antodoms I'll update this one as we have class this arvo, thanks man (-: